test/mitmproxy/data/addonscripts/recorder/error.py

```diff
@@ -0,0 +1,3 @@
+impotr recorder
```

Comment:
Let's make it more obvious that this is intended to be an invalid file with a syntax error, just so I don't accidentally fix it someday. :)

Reply:
Sure thankyou :) I will write this in the comments that this is an intended errors :)